src/main/kotlin/com/atlassian/performance/tools/report/api/result/EdibleResult.kt

```diff
@@ -26,17 +28,12 @@ class EdibleResult private constructor(
         }
     }
 
-    private val outlierTrimming = OutlierTrimming(
-        lowerTrim = 0.01,
-        upperTrim = 0.99
-    )
-
     val actionStats: InteractionStats by lazy {
         StatsMeter().measure(
             result = this,
             centralTendencyMetric = Mean(),
             dispersionMetric = StandardDeviation(),
-            outlierTrimming = outlierTrimming
+            outlierTrimming = OutlierTrimming(0.00, 1.0)
```

Comment:
Perhaps we should leave the `OutlierTrimming(0.01, 0.99)` for behavioral compatibility